src/Stryker.Core/Stryker.Core/Initialisation/InputFileResolver.cs

```diff
@@ -402,15 +144,15 @@ public string FindProjectUnderTest(IEnumerable<ProjectAnalyzerResult> testProjec
 
         public string FindProjectFile(string path)
```

Comment:
This should probably be moved to somewhere csharp and fsharp specific

Reply:
yes

Reply:
FindProjectFile is called before we know the language so I cannot move it to language specific code